### PR TITLE
Feature/mat 5805 special characters hr

### PR DESCRIPTION
**Pull request by nmorasb**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-5805](https://jira.cms.gov/browse/MAT-5805)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by jkotanchik-SB — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -18,8 +18,8 @@
 	<properties>
 		<java.version>17</java.version>
 		<springfox.version>3.0.0</springfox.version>
-		<cqframework.version>2.6.0</cqframework.version>
-		<hapi.fhir.r4.version>6.4.1</hapi.fhir.r4.version>
+		<cqframework.version>2.10.0</cqframework.version>
```
Comment (jkotanchik-SB) [inline pom.xml]: cql-elm-translator service is now using cqframework 2.11.0.

**Review by jkotanchik-SB — CHANGES_REQUESTED** [review]
Bump the cqframework to stay in sync with our cql-elm-translation service.

**Review by jkotanchik-SB — APPROVED** [review]
